src/api/app/jobs/send_event_emails.rb

```diff
@@ -1,10 +1,18 @@
-class SendEventEmails < CreateJob
-
-  attr_accessor :event
+class SendEventEmails
 
   def perform
-    subscribers = event.subscribers
-    return if subscribers.empty?
-    EventMailer.event(subscribers, event).deliver_later
+    Event::Base.where(mail_sent: false).order(:created_at).limit(100).each do |event|
+      self.mail_sent = true
+      begin
+        self.save
+      rescue ActiveRecord::StaleObjectError
+        # if someone else saved it too, better don't send continue
+        return
```

Comment:
because in that case two delayed jobs do the same thing and it's not worth to continue - but I'll add a comment